README.md

```diff
@@ -81,6 +82,46 @@ Converts components from synthetic shadow to native shadow.
 
 The only transformation it currently applies is to add the `static shadowSupportMode` property. Any other discrepancies between native shadow and synthetic shadow will have to be handled manually.
 
+### Template Directives
+
+#### Usage
+
+```sh
+lwc-codemod <template-directive-codemod> <path>
```

Comment:
This `<template-directive-codemod>` is a bit confusing. I would just leave the precise examples in each section, e.g.:

```
lwc-codemod multiple-if-true-if-false <path>
```

Alternatively, would it be easier for consumers if these were all folded into one codemod?